pabc.client/src/views/LoginView.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <div>
+  <section class="page">
```

Comment:
wat is je overweging om section te gebruiken? volgens mij doet section niks tenzij je een accessible name koppelt (bv met aria-labelledby)